src/runtime/config/configuration-qemu.toml.in

```diff
@@ -15,6 +15,7 @@
 path = "@QEMUPATH@"
 kernel = "@KERNELPATH@"
 image = "@IMAGEPATH@"
+# initrd = "@INITRDPATH@"
```

Comment:
@jpecholt, would you mind checking what happens if we start Kata Containers with `image` set but `initrd` set as `""`, and vice versa?

Depending on the results, we could have this one added uncommented.

Reply:
Both works

Reply:
Okay, that's good to know and I think leaving it as it's right now is the best approach.
Otherwise we have to force setting either image or initrd as empty during the build time.

Reply:
As it it's right now, as in, as it's in your patch. :-)